src/net_processing.cpp

```diff
@@ -5246,6 +5247,28 @@ void PeerManagerImpl::EvictExtraOutboundPeers(std::chrono::seconds now)
             }
         }
     }
+
+    // If we are at the max full outbound limit but not above, check if any peers
```

Comment:
nit f3b74d43823a038fb0aa4e0bcb7d4f99a7f90688:

The 2 logical paths depending on the value of `extra_outbound_count` could be more clear if this were an `if/else if` block.

Reply:
changed to `else if`